cicd/gitops/gitops-release-notes.adoc

```diff
@@ -23,4 +23,5 @@ For an overview of {gitops-title}, see xref:../../cicd/gitops/understanding-open
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
 
 // Modules included, most to least recent
+include::modules/gitops-release-notes-1-2.adoc[leveloffset=+1]
```

Comment:
@wtam2018 @Preeticp Links in line 26 and line 27 are broken. Are you already aware of it ?

Reply:
Hi @iam-veeramalla the links are not broken, if they were the build would not be successful. These include statements add the modules (small units of content) to a master assembly which generally includes related content or a complete user story with concepts procedures and references. See here for the rendered build: https://deploy-preview-35278--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes?utm_source=github&utm_campaign=bot_dp

Reply:
However @wtam2018 request you to add a blank line between the include statements please, like how it is done here: https://raw.githubusercontent.com/openshift/openshift-docs/main/cicd/pipelines/op-release-notes.adoc

Reply:
Awesome :)